csrc/device_lower/analysis/thread_predicate.cpp

```diff
@@ -200,6 +200,11 @@ ParallelTypeBitmap getReductionPredicateForUnusedParallelTypes(
 void ThreadPredicateMap::updateBitSet(const Expr* expr) {
   FUSER_PERF_SCOPE("GpuLower::Lower::ThreadPredicateMap::updateBitSet");
 
+  auto tv_out = ir_utils::getTvOutput(expr);
```

Comment:
Is this added since `GetMetaData` takes tensor inputs but produces non-tensor outputs?

Reply:
Yes!